pkgs/servers/immich/default.nix

```diff
@@ -0,0 +1,238 @@
+{ lib
+, pkgs
+, buildNpmPackage
+, fetchFromGitHub
+, fetchPypi
+, python
+, nodejs
+, nixosTests
+# build-time deps
+, pkg-config
+, makeWrapper
+, cmake
+# runtime deps
+, ffmpeg
+, imagemagick
+, libraw
+, vips
+}:
+let
+  buildNpmPackage' = buildNpmPackage.override { inherit nodejs; };
+  sources = lib.importJSON ./sources.json;
+  inherit (sources) version;
+
+  src = fetchFromGitHub {
+    owner = "immich-app";
+    repo = "immich";
+    rev = "v${version}";
+    inherit (sources) hash;
+  };
+
+  cli = buildNpmPackage' {
+    pname = "immich-cli";
+    inherit version;
+    src = "${src}/cli";
```

Comment:
Do you know what the difference from https://github.com/immich-app/CLI is?

Reply:
See comment above